### PR TITLE
Oscar - Creates Fixtures for Recommendation Requests (Updated to reflect new controller)

### LINKED ISSUE #47: Fixtures - RecommendationRequest

### PR DESCRIPTION
**Pull request by obenedek20**
Closes #47 
Merge after #66 - this incorporates the added user_id field inside the database from the new GET and POST controller and db changes PR

Adds a group of three and a single fixture for testing and story purposes with different statuses - professor_id has been added in place of professorEmail and professorName and user_id in place of requesterName to match requested changes by professor Conrad

### CODE REVIEW

**Review by divyanipunj — DISMISSED** [review]
LGTM!

**Comment by obenedek20** [comment]
It says Its waiting for UI Review but then when I click that it says there is nothing to review so I think it is all set

**Review by divyanipunj — DISMISSED** [review]
LGTM!

**Review by divyanipunj — DISMISSED** [review]

File: frontend/src/fixtures/recommendationRequestFixtures.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      status: "PENDING",
+      details: "test details",
+      recommendationTypes: "CS Department BS/MS program",
+    },
+  ],
+
+  threeRecommendations: [
+    {
+      id: 2,
+      professorName: "testname1",
```
Comment (pconrad) [inline frontend/src/fixtures/recommendationRequestFixtures.js]: Similar to comments above.

**Review by pconrad — CHANGES_REQUESTED** [review]
This needs to be adjusted based on the changes to the RecommendationRequest entity and CRUD operations in #39 , so let's hold off until that one is final (or at least nearing it's final form.)

Until then, we won't have a clear idea of what the fields should look like here.

**Review by divyanipunj — DISMISSED** [review]
LGTM!

File: frontend/src/fixtures/recommendationRequestFixtures.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    {
+      id: 1,
+      professorName: "testname",
+      professorEmail: "testemail@ucsb.edu",
+      requesterName: "recname",
+      submissionDate: "2022-02-02T12:00",
+      completionDate: "",
+      status: "PENDING",
+      details: "test details",
+      recommendationTypes: "CS Department BS/MS program",
```
Comment (pconrad) [inline frontend/src/fixtures/recommendationRequestFixtures.js]:
```suggestion
      recommendationType: "CS Department BS/MS program",
```

File: frontend/src/fixtures/recommendationRequestFixtures.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    {
+      id: 2,
+      professorName: "testname1",
+      professorEmail: "testemail1@ucsb.edu",
+      requesterName: "recname1",
+      submissionDate: "2022-02-02T12:00",
+      completionDate: "",
+      status: "PENDING",
+      details: "test details",
+      recommendationTypes: "CS Department BS/MS program",
```
Comment (pconrad) [inline frontend/src/fixtures/recommendationRequestFixtures.js]:
```suggestion
      recommendationType: "CS Department BS/MS program",
```

File: frontend/src/fixtures/recommendationRequestFixtures.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+    {
+      id: 3,
+      professorName: "testname2",
+      professorEmail: "testemail2@ucsb.edu",
+      requesterName: "recname2",
+      submissionDate: "2022-02-02T12:00",
+      completionDate: "2022-02-02T12:00",
+      status: "COMPLETED",
+      details: "test details",
+      recommendationTypes: "MS program (other than CS Dept BS/MS)",
```
Comment (pconrad) [inline frontend/src/fixtures/recommendationRequestFixtures.js]:
```suggestion
      recommendationType: "MS program (other than CS Dept BS/MS)",
```

File: frontend/src/fixtures/recommendationRequestFixtures.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    {
+      id: 4,
+      professorName: "testname3",
+      professorEmail: "testemail3@ucsb.edu",
+      requesterName: "recname3",
+      submissionDate: "2022-02-02T12:00",
+      completionDate: "2022-02-02T12:00",
+      status: "DENIED",
+      details: "test details",
+      recommendationTypes: "Scholarship or Fellowship",
```
Comment (pconrad) [inline frontend/src/fixtures/recommendationRequestFixtures.js]:
```suggestion
      recommendationType: "Scholarship or Fellowship",
```

**Review by pconrad — CHANGES_REQUESTED** [review]
there's only one recommendationType per request, so this should be singular.

**Comment by obenedek20** [comment]
professor_id has been added in place of professorEmail and professorName and user_id in place of requesterName to match requested changes by professor Conrad as well as changing recommendationTypes to recommendationType

**Review by pconrad — APPROVED** [review]
LGTM!

**Pull request merged**